dev/io.openliberty.accesslists.internal/src/com/ibm/ws/tcpchannel/internal/FilterListFastStr.java

```diff
@@ -319,9 +319,9 @@ public int getCurrentSize() {
         /**
          * get the array of hashcodes for the substrings that represent this entry
          * 
-         * @return the array of hashcodes for the substrings that represent this
-         *         entry
+         * @return the array of hashcodes for the substrings that represent this entry
          */
+        @Trivial
```

Comment:
why is trivial needed here, since instrumentation is disabled for this bundle - just for reading clarity?

Reply:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/2861921/144472768-2b32c5a3-2e63-4b2b-8cfc-0d459c1c929b.png">

Reply:
It came from the io.openliberty.netty.internal.impl version. I did not spot that such implementation being disabled would make this an no-op. How best do you think this is resolved @wtlucy  ?

Reply:
I see - now that you point out this came from the netty bundle, I recall that I added the `@Trivial` there because instrumentation isn't disabled for that bundle and the logging for these methods was excessive! I think we should leave instrumentation disabled for this accesslists bundle, because there's a small chance it will have performance implications for code currently using the channelfw. But now that you've given me the history, I'm fine with leaving the `@Trivial` in place, in case we decide later on that instrumentation _can_ be safely enabled.